lib/network/modules/InteractionHandler.js

```diff
@@ -165,8 +165,11 @@ class InteractionHandler {
 
   onContext(event) {
     let pointer = this.getPointer({x:event.clientX, y:event.clientY});
-    this.checkSelectionChanges(pointer, event);
-    this.selectionHandler._generateClickEvent('oncontext', event, pointer);
+    const hoveredElements = {
+      nodes: this.selectionHandler.getNodeAt(pointer.DOM),
+      edges: this.selectionHandler.getEdgeAt(pointer.DOM)
```

Comment:
There is no DOM object within pointer, just x and y. Use the pointer object directly.

`nodes` and `edges` are arrays, please add any hovered over nodes and edges as array, and *only if these are present*; i.e. `getNodeAt()` and `getEdgeAt()` may return `undefined`.